packages/react-router/tests/useRoutes.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { type RootRoute } from "../src/routeRegistry.ts";
+import { Runtime } from "../src/runtime.ts";
+import { RuntimeContext } from "@squide/core";
+import { renderHook, type RenderHookOptions } from "@testing-library/react";
+import { useRoutes } from "../src/useRoutes.ts";
+import { type ReactNode } from "react";
+
+function renderWithRuntime<TProps>(runtime: Runtime, additionalProps: RenderHookOptions<TProps> = {}) {
```

Comment:
can't renderWithRuntime have an optional runtime? all the tests only do new Runtime, can we make it optional?

Reply:
What would be the point of doing this? The function is quite useless without a runtime, the reason for the function to exist is to setup the runtime.